CompilerSource/compiler/compile.cpp

```diff
@@ -160,14 +160,27 @@ dllexport void ide_handles_game_launch() { run_game = false; }
 static bool redirect_make = true;
 dllexport void log_make_to_console() { redirect_make = false; }
 
+bool ends_with(std::string const &fullString, std::string const &ending) {
+    if (fullString.length() < ending.length())
+      return false;
+
+    return (0 == fullString.compare (fullString.length() - ending.length(), ending.length(), ending));
```

Comment:
This is a pretty clever implementation, actually. Quite efficient.

Reply:
Thanks 😄